kingfisher_scrapy/base_spider.py

```diff
@@ -76,6 +76,7 @@ class BaseSpider(scrapy.Spider):
 
     ocds_version = '1.1'
     date_format = 'date'
+    date_required = None
```

Comment:
```suggestion
    date_required = False
```
and could you document this new attribute, please?